pkg/status/status.go

```diff
@@ -52,14 +52,14 @@ func GetStatus() (map[string]interface{}, error) {
 	stats["platform"] = platformPayload
 	stats["hostinfo"] = host.GetStatusInformation()
 	now := time.Now()
-	stats["time"] = now.Format(timeFormat)
+	stats["time"] = now.Format(render.TimeFormat)
 
 	stats["JMXStatus"] = GetJMXStatus()
 
 	stats["logsStats"] = logs.GetStatus()
 
 	if config.Datadog.GetBool("cluster_agent.enabled") {
-		stats["clusterAgentStatus"] = getDCAStatus()
+		stats["clusterAgentStatus"] = clusteragent.GetStatus()
```

Comment:
This `clusteragent.GetStatus()` is the client-side status which lives in the `util/clusteragent` package.